api/src/nlp/services/nlp-entity.service.spec.ts

```diff
@@ -48,6 +53,12 @@ describe('nlpEntityService', () => {
         NlpValueService,
         NlpValueRepository,
         NlpSampleEntityRepository,
+        {
+          provide: CACHE_MANAGER,
+          useValue: {
+            del: jest.fn(),
```

Comment:
```suggestion
            get: jest.fn(),
            set: jest.fn(),
```

Hi @MohamedAliBouhaouala, as discussed this update can resolve this error
![image](https://github.com/user-attachments/assets/e910b709-88b9-447b-b167-d74b6eff8a0f)